tests/ParsingHelperTest.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+use viget\videoembed\enums\VideoType;
+use viget\videoembed\helpers\ParsingHelper;
+
+final class ParsingHelperTest extends TestCase
+{
+    public function testGetVideoTypeFromUrl(): void
+    {
+        $this->assertEquals(
+            VideoType::YOUTUBE,
+            ParsingHelper::getVideoTypeFromUrl('https://www.youtube.com')
+        );
```

Comment:
Great!

These are the two NPF test URLs:
- https://youtu.be/--HXLM8GuxA?si=pNahKJLszKr8J00u
- https://www.youtube.com/watch?v=--HXLM8GuxA